test/unit/alphabet/nucleotide/rna4_test.cpp

```diff
@@ -7,10 +7,14 @@
 
 #include "../alphabet_test_template.hpp"
 #include "../alphabet_constexpr_test_template.hpp"
+#include "../semi_alphabet_test_template.hpp"
+#include "../semi_alphabet_constexpr_test_template.hpp"
```

Comment:
just seeing i here but it concerns all files. The ALPHABETICAL ORDER 😱 💥  🤣

Reply:
thanks for spotting. good thing editors have regex support ;)